cypress/e2e/index.cy.ts

```diff
@@ -451,12 +344,12 @@ describe("Case list", () => {
       ]
       cy.task("insertCourtCasesWithNotes", { caseNotes: caseNotes, force: "01" })
 
-      loginAndGoToUrl()
+      cy.login("bichard01@example.com", "password")
+      cy.visit("/bichard")
 
-      cy.get("tr").not(":first").eq(0).get("td:nth-child(5)").contains(`Case00000`)
-      cy.get("tr").not(":first").eq(0).get("td:nth-child(7)").should("be.empty")
-      cy.get("tr").not(":first").eq(1).get("td:nth-child(7)").contains(`1`).should("exist")
-      cy.get("tr").not(":first").eq(2).get("td:nth-child(7)").contains(`3`).should("exist")
+      cy.get("tr").not(":first").eq(1).get("td:nth-child(7)").contains(`Preview`).should("exist")
```

Comment:
I think that `cy.get("tr").not(":first").eq(1).get("td:nth-child(7)").contains(`Preview`)` is redundant. The following line `cy.get("tr").not(":first").eq(1).get("td:nth-child(7)").contains(`Preview`).trigger("click")` does this check too. If `cy.get("tr").not(":first").eq(1).get("td:nth-child(7)").contains(`Preview`)` was not there, cypress would let us know.

Reply:
I think having the `should("exist")` is a more explicit check, rather than cypress trying to find the right box to click- what do you think?

Reply:
I'm not super fussed, if you think that `should("exist")` makes it easier to read, I'm happy to leave it. Personally, I'd remove it, the file is pretty long anyway.